src/components/marker/Marker.jsx

```diff
@@ -16,8 +16,8 @@ export default class Marker extends React.Component {
                 viewBox="0 -5 10 10"
                 refX={this.props.refX}
                 refY="0"
-                markerWidth="6"
-                markerHeight="6"
+                markerWidth={this.props.markerWidth || "6"}
```

Comment:
The fallback `"6"`, should be declared in `graph.config.js`, there together with this value you should also add the proper documentation for the new properties `markerWidth` and `markerHeight`. Just check how the other props are set up in that same file and try to follow the same pattern.

Reply:
@danielcaldas I have declared the new props in config file and added some documentation. Let me know if something else is missing.